packages/common/base/src/types/Crossmint.ts

```diff
@@ -1,5 +1,22 @@
 import { type ValidateAPIKeyPrefixExpectations, validateAPIKey } from "@/apiKey";
 
+export type FarcasterMetadata = {
```

Comment:
As these are Auth specific things, shouldn't they go in a separate file? Would also be good to have all Auth types in a single place shared between SDK and crossmint-nextjs